### PR TITLE
Handle alphanumeric class names in `FilePath` cop

### PR DESCRIPTION
**Pull request by andyw8**

### CODE REVIEW

**Comment by nijikon** [comment]
@andyw8 please rebase on top of master.

**Comment by andyw8** [comment]
@nijikon done.

File: spec/rubocop/cop/rspec/file_path_spec.rb [hunk 1]
```diff
@@ -109,6 +109,13 @@
     expect(cop.offenses).to be_empty
   end
 
+  it 'handles alphanumeric class names' do
+    inspect_source(cop,
```
Comment (nijikon) [inline spec/rubocop/cop/rspec/file_path_spec.rb]: Can you please format this like other lines in the file?

Comment (andyw8) [inline spec/rubocop/cop/rspec/file_path_spec.rb]: @nijikon done.

**Pull request merged**